doc/installation.rst

```diff
@@ -77,77 +56,69 @@ __ http://www.macports.org/ports.php?by=name&substr=spyder
-   
+
      To fix it you will have to set these environment variables in your
      ``~/.profile`` (or ``~/.bashrc``) manually::
-        
+
         export LANG=en_US.UTF-8
         export LC_ALL=en_US.UTF-8
 
 |
 
-Installing on Linux
--------------------
-
-The easy way
-~~~~~~~~~~~~
-
-Thanks to the Spyder team and `Anaconda <https://www.anaconda.com/>`_, Spyder is included
-in the `Anaconda <https://www.anaconda.com/download/>`_ Python distribution,
-which comes with everything you need to get started in an all-in-one package.
-
-
-The harder way
-~~~~~~~~~~~~~~
+GNU/Linux
+~~~~~~~~~
 
 Please refer to the `Requirements`_ section to see what other packages you
 might need.
 
 **Ubuntu**:
 
-   * Using the official package manager: ``sudo apt-get install spyder``.
+Using the official package manager: ``sudo apt-get install spyder``.
 
      .. note::
 
         This package could be slightly outdated. If you find that is the case,
         please use the Debian package mentioned below.
 
-   * Using the `pip <https://pypi.python.org/pypi/pip/>`_ package manager:
-
-     * Installing: ``sudo pip install spyder``
-     * Updating: ``sudo pip install -U spyder``
 
-     
 **Debian Unstable**:
-  
-   Using the package manager: ``sudo apt-get install spyder``
 
-   The Spyder's official Debian package is available `here`__ 
-  
-   __ http://packages.debian.org/fr/sid/spyder.
+Using the package manager: ``sudo apt-get install spyder``
+
+The Spyder's official Debian package is available `here`__
+
+__ http://packages.debian.org/fr/sid/spyder.
 
 
 **Other Distributions**
 
-   Spyder is also available in other GNU/Linux distributions, like
+Spyder is also available in other GNU/Linux distributions, like
 
-   * `Archlinux <https://aur.archlinux.org/packages/?K=spyder>`_
+* `Archlinux <https://aur.archlinux.org/packages/?K=spyder>`_
 
-   * `Fedora <https://admin.fedoraproject.org/pkgdb/acls/name/spyder?_csrf_token=ab2ac812ed6df3abdf42981038a56d3d87b34128>`_
+* `Fedora <https://admin.fedoraproject.org/pkgdb/acls/name/spyder?_csrf_token=ab2ac812ed6df3abdf42981038a56d3d87b34128>`_
 
-   * `Gentoo <http://packages.gentoo.org/package/dev-python/spyder>`_
+* `Gentoo <http://packages.gentoo.org/package/dev-python/spyder>`_
 
-   * `openSUSE <https://build.opensuse.org/package/show?package=python-spyder&project=home%3Aocefpaf>`_
+* `openSUSE <https://build.opensuse.org/package/show?package=python-spyder&project=home%3Aocefpaf>`_
 
-   * `Mageia <http://mageia.madb.org/package/show/name/spyder>`_
+* `Mageia <http://mageia.madb.org/package/show/name/spyder>`_
 
-   Please refer to your distribution's documentation to learn how to install it
-   there.
+Please refer to your distribution's documentation to learn how to install it
+there.
 
 |
 
-Installing or running directly from source
-------------------------------------------
+
+The Expert Way: Installing or running directly from source
```

Comment:
I think this should be

> The Experts Way: Installing with pip

Reply:
Changed head per my previous comment and subhead per yours here.